README.md

```diff
@@ -2,4 +2,45 @@
 
 A simple example showing how to setup a 3d scene across windows on the same origin using three.js and localStorage. Code should be self explanatory
 
-follow me on twitter at [@_nonfigurativ_](https://twitter.com/_nonfigurativ_) for more stuff like this
\ No newline at end of file
+follow me on twitter at [@_nonfigurativ_](https://twitter.com/_nonfigurativ_) for more stuff like this
+
+
+
+## HOW TO RUN THE PROJECT
+
+#### 1) Install http-server globaly:
+-------------
```

Comment:
* why adding a breakline after this H4 (4 #) ?
* Why use H4 when above you have a H2 (2 #) ?